docs/book/toc.md

```diff
@@ -167,6 +166,7 @@
   * [Kubernetes](how-to/popular-integrations/kubernetes.md)
   * [MLflow](how-to/popular-integrations/mlflow.md)
   * [Skypilot](how-to/popular-integrations/skypilot.md)
+  * [Implement a custom integration](how-to/stack-deployment/implement-a-custom-integration.md)
```

Comment:
not sure this is the right place for it ... how about adding a new HOw-to called "Contributing to ZenML" that points to https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md and has a page fr writing and contributing an integration?

Reply:
done